### DISCUSSION TITLE
Spike: Research captchas to use for communities join screens

### DISCUSSION THREAD

**Discussion by J-Son89** [status-im/status-legacy#16657] [General]
See the designs:
https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?node-id=7643%3A608085&t=bc5QBHpQ3uZT8W6b-0

There is the use of a captcha in the designs for the communities join screens. 

It is still unclear what Captcha service we want to use. Research some possible solutions, including:

https://www.hcaptcha.com/
also consider an in house solution.

Definition of done:
Research possible solutions
Document these solutions
Bring forward solutions to team and analyse the pros/cons

**Comment by J-Son89** [comment]
https://github.com/hCaptcha/react-native-hcaptcha

**Comment by du82** [comment]
Captchas will not solve the spam problem, because the spammers are using a forked desktop client with automation tools. All captchas do is make the app less private, and more annoying for the average user.

All captchas will do is make using Status over LAN impossible, and make it easier for censors to discover users.

If we *must* have captchas, why not make it optional and let individual communities choose if they want to enable it?

**Comment by J-Son89** [comment]
There was an agreement to postpone this research until post mvp
